markdowns/workingGroup/current.md

```diff
@@ -1,23 +1,21 @@
 # Interchain NFT + Metadata Current Agenda
 
-## Next Call: Working Group Call #10
-
-### Thursday, 4 March 2021
+## Next Call: Metadata Subgroup Call H
+### Thursday, 11 March 2021
 
 ### Call in details: 
   1. Time: 8am PT, 4pm GMT, 5pm CET
   2. Join: https://zoom.us/j/9990000911 Password: Pandora   
-### AGENDA
-1. Roadmap and progress
-1. The Token Taxonomy - Interwork Alliance
-1. Interchain Token Templates
-1. Github PRs & Issues
-
+### THIS WEEK
```

Comment:
```suggestion
### AGENDA
```
Not sure if this is changed on purpose. If not, let's stick with the original structure.